8_visualize_metrics.yml

```diff
@@ -0,0 +1,220 @@
+                metric_file = '4_evaluation/out/exceedance_metrics_by_seg.csv',
+                metric_col = I('true_neg'),
+                plot_model = I('sntemp_full'),
+                legend_text = I('SNTEMP-Model Exceedance (T-Neg)'))
+
+# exceedance for rgnc model
+  8_visualize/out/rgnc2_full_false_pos_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/exceedance_metrics_by_seg.csv',
+                metric_col = I('false_pos'),
+                plot_model = I('rgcn2_full_full'),
+                legend_text = I('RGNC-Model Exceedance (F-Pos)'))
+
+  # exceedance for sntemp model
+  8_visualize/out/sntemp_full_false_pos_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/exceedance_metrics_by_seg.csv',
+                metric_col = I('false_pos'),
+                plot_model = I('sntemp_full'),
+                legend_text = I('SNTEMP-Model Exceedance (F-Pos)'))
+
+  # exceedance for rgnc model
+  8_visualize/out/rgnc2_full_false_neg_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/exceedance_metrics_by_seg.csv',
+                metric_col = I('false_neg'),
+                plot_model = I('rgcn2_full_full'),
+                legend_text = I('RGNC-Model Exceedance (F-Neg)'))
+
+  # exceedance for sntemp model
+  8_visualize/out/sntemp_full_false_neg_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/exceedance_metrics_by_seg.csv',
+                metric_col = I('false_neg'),
+                plot_model = I('sntemp_full'),
+                legend_text = I('SNTEMP-Model Exceedance (F-Neg)'))
+
+  # max temps for rgnc model
+  8_visualize/out/rgnc2_full_max_temp_error_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/max_metrics_by_seg_year.csv',
+                metric_col = I('error_obs_pred'),
+                plot_model = I('rgcn2_full_full'),
+                legend_text = I('RGNC-Model Max Temps Error'))
+
+  # max temps for sntemp model
+  8_visualize/out/sntemp_full_max_temp_error_on_network.png:
+    command: plot_metric_map(out_file = target_name,
+                network_ind = '1_network/out/network.rds.ind',
+                metric_file = '4_evaluation/out/max_metrics_by_seg_year.csv',
+                metric_col = I('error_obs_pred'),
+                plot_model = I('sntemp_full'),
+                legend_text = I('SNTEMP-Model Max Temps Error'))
```

Comment:
You can drop the model name and "model" out of these legends. The file name will help keep track, and that will give the user flexibility with how the want the labels to appear.

Reply:
Legend suggestion: "Max temp error (deg C)"